profile_collection/startup/csx1/startup/darkframes.py

```diff
@@ -85,14 +122,18 @@ def dark_plan(gain, num_dark_images):
 
 #for gain in (1, 2, 8):
 #for gain in (0, 1, 2):
+#for gain in (0, 1):  #TODO for testing multiple darks
 for gain in (0,):
     # 1, 2, and 8 are the multipliers.  auto = 8,
     #gain_bit_dict = {0:'auto', 1: 'x2', 2: 'x1'} #where the keys are the epics values
-    NUM_DARK_IMAGES = 10
-    this_dark_plan = functools.partial(dark_plan, gain, NUM_DARK_IMAGES)
+    this_dark_plan = functools.partial(dark_plan, gain)
+
+    dark_frame_preprocessor = TemporarilyCustomizedDarkFrameProcessor(
+        dark_plan=this_dark_plan, detector=fccd, max_age=600,
+        locked_signals=(fccd.cam.acquire_time,),
+        stream_name=f'dark')
```

Comment:
I still find the stream name "dark" too vague. I propose "dark_frames".

Reply:
That's fine with me---always one to prefer verbose and specific to succinct and vague---but I think we should poll @DanOlds, @EliotGann, @ambarb. To be clear, this is something that beamlines can vary on if they like with little cost, but it might simplify things to have a consistent convention we apply as a facility unless there is good reason to diverge.

Reply:
'dark_images' instead?  are all of the key's for image data `[camera]_images`, e.g., `fccd_images`.  

so then this would extend to the flat field, but depending on how it is collected the last part might not be `images`.  At CSX, we discussed the need to fully process the data and then have one single image that represents that flat field (dark subtracted and gain correct). Choices
* `flatfield_correction`
* `flatfield_image` (as opposed to the data for flat field, which would be `flatfield_images`)
* `flatfield_array`
*`flatfield_analysis`

Reply:
Let's continue this at https://github.com/bluesky/bluesky-darkframes/issues/22